src/PrestaShopBundle/Form/Admin/Configure/AdvancedParameters/Webservice/WebserviceKeyType.php

```diff
@@ -111,6 +112,13 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                             'Admin.Advparameters.Notification'
                         ),
                     ]),
+                    new Regex([
+                        'pattern' => '/^[a-zA-Z0-9@\#\?\-\_]+$/i',
+                        'message' => $this->trans(
+                            'The key can contains only numbers, letters, and these special chars: @ ? # - _',
```

Comment:
```suggestion
                            'Only letters, numbers, and the following special characters are allowed: %allowed_characters%',
```

Reply:
Thanks Julie!
I've applied your suggestion through a new commit, because the suggested change is missing the parameter in the code to show the special characters 😊